src/Validation/ValidatorInterface.php

```diff
@@ -19,10 +19,11 @@ interface ValidatorInterface
 	 * Validate params.
 	 *
 	 * @param array<string, mixed> $data Date to check from reference helper.
+	 * @param bool $strictValidation Is falidation is strict.
```

Comment:
```suggestion
	 * @param bool $strictValidation Is falidation is strict.
```
I don't understand this phrasing 😄